Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -56,6 +56,7 @@ static void InitAVCodec()
   if (first_run)
   {
     av_register_all();
+    avformat_network_init();
```

Comment:
I don't know about Chromecast, but Twitch uses RTMP. All you have to do is set the DumpPath to something like "rtmp://live.twitch.tv/app/...".